api/api_public_blockchain.go

```diff
@@ -150,6 +150,34 @@ func (s *PublicBlockChainAPI) GetAccount(ctx context.Context, address common.Add
 	return serAcc, state.Error()
 }
 
+// GetHeaderByNumber returns the requested canonical block header.
+// * When blockNr is -1 the chain head is returned.
+// * When blockNr is -2 the pending chain head is returned.
+func (s *PublicBlockChainAPI) GetHeaderByNumber(ctx context.Context, number rpc.BlockNumber) (map[string]interface{}, error) {
+	header, err := s.b.HeaderByNumber(ctx, number)
+	if header != nil && err == nil {
+		response, err := s.rpcOutputHeader(header)
+		if number == rpc.PendingBlockNumber {
```

Comment:
Unlike ethereum, we do not support pending block number. So, I think this if clause is not needed. (HeaderByNumber will return an error `ErrPendingBlockNotSupported`.

Reply:
I reflected it. Thank you always!